### PR TITLE
alsa-state: Clean up application of machine-specific settings

### PR DESCRIPTION
**Pull request by gunnarx**
[GDP-706] Make alsa-state bbappend more consistent

Seems to build OK for minnowboard.  I have not build-tested for other platforms (PR check will do).   
Also needs on target functionality testing before merge.

### CODE REVIEW

File: meta-genivi-dev/poky/meta/recipes-bsp/alsa-state/alsa-state.bbappend [hunk 1]
```diff
@@ -1,5 +1,2 @@
-FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
+FILESEXTRAPATHS_prepend := ":${THISDIR}/${PN}"
```
Comment (DunderRoffe) [inline meta-genivi-dev/poky/meta/recipes-bsp/alsa-state/alsa-state.bbappend]: As this is a prepend the colon should be at the end. Yocto is warning about this when building for r-car-m3-starter-kit

Comment (gunnarx) [inline meta-genivi-dev/poky/meta/recipes-bsp/alsa-state/alsa-state.bbappend]: Makes sense

Comment (frznlogic) [inline meta-genivi-dev/poky/meta/recipes-bsp/alsa-state/alsa-state.bbappend]: This has been fixed it seems so I'm overriding @DunderRoffe s rejection

**Comment by frznlogic** [comment]
This has been tested on r-car now as well, and sound seems to work nicely. rebasing and merging the commit.

**Pull request merged**